src/containers/ProjectContainer.js

```diff
@@ -124,18 +124,32 @@ const ProjectContainer = React.createClass({
 
   fetchPullRequests() {
     let { gitHubToken, selectedRepo } = getSettings();
-    return $.ajax({
-      url: `https://api.github.com/repos/${selectedRepo}/pulls?state=open&access_token=${gitHubToken}`,
-      method: 'GET'
-    }).then(data =>
-      _.map(data, pullRequest => (
-        {
-          id: pullRequest.id,
-          url: pullRequest.html_url,
-          commitsUrl: pullRequest.commits_url
-        }
-      ))
-    ).fail(() => this.setState({ error: true }));
+    return new Promise(resolve => {
+      if (selectedRepo !== null && selectedRepo !== undefined) {
+        $.ajax({
+          url: `https://api.github.com/repos/${selectedRepo}/pulls?state=open&access_token=${gitHubToken}`,
+          method: 'GET'
+        }).then(data =>
+          resolve(
+            _.map(data, pullRequest => (
+              {
+                id: pullRequest.id,
+                url: pullRequest.html_url,
+                commitsUrl: pullRequest.commits_url
+              }
+            ))
+          )
+        ).fail(() => { 
+          this.setState({ error: true }); 
+          resolve(); 
+        });
+      } else {
+        resolve();
+      }
+    });
+
+  
+
```

Comment:
Hi Dylan,
For some reason, Sublime is not warning me about extra white space, so I'm
not sure what white space you mean.
Also, assuming I was able to fix it, what is the right protocol afterwards?
Would I submit a new pull request?

On Fri, Mar 18, 2016 at 2:33 PM, Dylan Kirby <notifications@github.com>
wrote:

> In src/containers/ProjectContainer.js
> <https://github.com/mojotech/pivotal-swimlanes/pull/104#discussion_r56702551>
> :
>
> > +                url: pullRequest.html_url,
> > +                commitsUrl: pullRequest.commits_url
> > +              }
> > +            ))
> > +          )
> > +        ).fail(() => {
> > +          this.setState({ error: true });
> > +          resolve();
> > +        });
> > +      } else {
> > +        resolve();
> > +      }
> > +    });
> > +
> > +
> > +
>
> Hey @tbarmann <https://github.com/tbarmann>, could you just take this
> extra whitespace out?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/mojotech/pivotal-swimlanes/pull/104/files/1187600f972589ba330e29ecb1957e0101cfe7d1#r56702551>
>